analyses/fusion_filtering/results/pbta-fusion-recurrent-fusion-bysample.tsv

```diff
@@ -318,7 +319,7 @@ BS_B7GEH89C	0	0	0	1	0
 BS_B7QS4DHK	0	0	0	1	0
 BS_B7X8VXQ3	0	0	1	0	0
 BS_B8T7M0WV	0	0	0	1	0
-BS_B9V8RGTA	0	0	0	1	0
+BS_B9V8RGTA	0	1	0	0	0
```

Comment:
It's not obvious to me that we would expect this change?

Reply:
This seems to not have been correct in v22, but not sure why. In both v22 and v23 putative oncogenic, we see: 

```r
# A tibble: 4 × 5
  Sample      FusionName  LeftBreakpoint RightBreakpoint Fusion_Type
  <chr>       <chr>       <chr>          <chr>           <chr>      
1 BS_B9V8RGTA EWSR1--FLI1 22:29287134    11:128805366    in-frame   
2 BS_B9V8RGTA EWSR1--FLI1 22:29287134    11:128805366    in-frame   
3 BS_B9V8RGTA EWSR1--FLI1 22:29287134    11:128807180    in-frame   
4 BS_B9V8RGTA EWSR1--FLI1 22:29288786    11:128805366    frameshift 
```